userspace/libsinsp/container_info.h

```diff
@@ -115,10 +123,13 @@ class sinsp_container_lookup
 	 */
 	short delay()
 	{
-		return 125 << (m_retry-1);
+		short curr_delay = 125 << (m_retry-1);
```

Comment:
I have a fear of a `short` overflowing for `m_retry==10`
```suggestion
		unsigned int curr_delay = 125 << (m_retry-1);
```

Reply:
at the moment it's not configurable so it's less important, but yes good idea.